### PR TITLE
21 reorganize the file structure based on httpsgithubcomttimbersbreast cancer predictor pytree001

### PR DESCRIPTION
**Pull request by zywkloo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,11 @@ All required dependencies are listed in this [conda environment file](environmen
 
 1. (Suggested) Create a new conda environment based on the provided [YAML file](environment.yaml).
   - Run `conda env create --name avalon --file=environment.yaml`
+  - Install the ipykernel:
+```
+conda activate avalon
+python -m ipykernel install --user --name avalon --display-name "Python (avalon)"
+```
   - Then switch to `avalon` env by clicking the drop-down, and select `avalon`
```
Comment (zywkloo) [inline README.md:29]: What's this part for , may I ask?

**Review by zywkloo — COMMENTED** [review]
conda activate avalon
python -m ipykernel install --user --name avalon --display-name "Python (avalon)"

May I ask what this line is doing?

File: README.md (hunk 1 above)
Comment (SoloSynth1) [reply README.md:29]: We have to run this in order for Jupyter Lab to recognize the kernel of the new conda environment

Comment (SoloSynth1) [reply README.md:29]: Here is the documentation on this:
https://ipython.readthedocs.io/en/stable/install/kernel_install.html

Comment (zywkloo) [reply README.md:29]: Ok, nice. I usually just refresh the web page. thank you for explanation.

Comment (SoloSynth1) [reply README.md:29]: OK thanks, I don't know why but my jupyter lab recognize the kernel in this way only, the reloading does not work for me.

**Review by MoNorouzi23 — APPROVED** [review]

**Review by MoNorouzi23 — APPROVED** [review]

**Pull request merged**